packages/editor/src/systems/EditorControlSystem.ts

```diff
@@ -365,7 +365,7 @@ const execute = () => {
         selectedParentEntity === clickStartEntity ? closestIntersection.entity : selectedParentEntity
 
       // If hiding children of GLB, don't allow those children to be selected (clicking in scene view)
-      if (hierarchyFeatureFlagEnabled && selectedParentEntity) {
+      if (!showGlbChildrenFeatureFlagEnabled && selectedParentEntity) {
```

Comment:
this can just be simplified to `getState(EditorHelperState).showGlbChildren` rather than persisting a variable